best-practices/native/nativescript/evironment.md

```diff
@@ -0,0 +1,78 @@
+The best prectice recomends not use more than 2 properties, production and apiUrl properties.
+To achive that, we need to make some chagens on environment files as you can see below:
+
+1 - This goes to environment.<dev|stage|prod>.ts
+<export const environment = {
+    production: true,
+    apiUrl: 'http://www.yoursite.com'
+};>
+
+2 - On environment.tns.ts:
+
+<import { environment as devEnvironment } from './environment.dev';
+import { environment as prodEnvironment } from './environment.prod';
+// TODO: import additional environment (e.g. uat) if required.
+export const environment = (() ={
+  let envVars;
+  if (Object.prototype.hasOwnProperty.call(process.env, 'environment') &&
+        process.env.environment
+  ) {
+    switch (process.env.environment) {
+      case 'prod':
+        envVars = prodEnvironment;
+        break;
+      // TODO: Add additional environment (e.g. stage) if required. 
+      default:
+        envVars = devEnvironment;
+    }
+  } else {
+    envVars = devEnvironment;
+  }
+  return envVars;
+})();>
```

Comment:
Please put all code blocks inside of triple back ticks:

<pre>```
function sampleCodeBlock() {
  ...
}
```</pre>

Reply:
Got it. Done